apps/docs/src/app/core/component-docs/time-picker/time-picker-header/time-picker-header.component.html

```diff
@@ -1,17 +1,41 @@
-    Library for Angular Time component. <br /><br />
-    The time picker supports the same keyboard navigability as the time component. The calendar can be opened by
-    focusing the time picker input field and closed by focusing the calendar icon button and pressing enter or space.
-    <br /><br />
-    The Time Picker component <strong>relies on provided datetime implementation</strong> (<code>DatetimeAdapter</code>)
-    and <strong>datetime formats</strong> (<code>DateTimeFormats</code>).
-    <br />
-    Currently fundamental-ngx offer 2 kind of adapters: FdDateTimeAdapter and
-    <a fd-link routerLink="/core/moment-datetime-adapter">MomentDateTimeAdapter</a>. <br /><br />
-    The Time Picker component uses Time Component internally, for internationalization configuration, check
-    <code>Internationalization</code> section in <code>Time</code> documentation
+<description>
+    <p>
+        The Time Picker component allows the user to easily set a time using either an HTML5 time input or the
+        Fundamental Library for Angular Time component.
+        <br /><br />
+        The time picker supports the same keyboard navigability as the time component. The calendar can be opened by
+        focusing the time picker input field and closed by focusing the calendar icon button and pressing enter or
+        space.
+        <br /><br />
+        The Time Picker component uses Time Component internally, for internationalization configuration, check
+        <code>Internationalization</code> section in <code>Time</code> documentation
+    </p>
+    <fd-message-strip [dismissible]="false" type="information">
+        <b>IMPORTANT</b>
+        <br />
+        The Time Picker component relies on provided datetime implementation (DatetimeAdapter) and datetime formats
+        (DateTimeFormats).
+        <br />
+        The current examples are based on FdDatetimeAdapter.
+        <br />
+        The FdDatetimeAdapter is based on native date api (formatting is based on
+        <a href="https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl/DateTimeFormat">
+            Intl object </a
+        >, parsing is based on
+        <a href="https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date/parse">
+            Date.parse() </a
+        >).
+        <br />
+        Since this adapter has some disadvantages, formatting may vary among browsers and parsing is limited to en
+        locale only.
+        <br />
+        You can use a custom fundamental adapter based on a third party library:
+        <a href="https://momentjs.com/" target="_blank">https://momentjs.com/</a>.
+        <br />
+        Please check examples and documentation on:
+        <b> <a routerLink="/core/moment-datetime-adapter" target="_blank"> Moment date time adapter </a> </b>.
+    </fd-message-strip>
```

Comment:
can we keep it in a sharable component so we don't need to repeat it on every date/time-related pages?